src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowFlag.java

```diff
@@ -20,6 +20,7 @@ public enum FlowFlag {
     LIP_CASE,
     CLAIM_ISSUE_HWF,
     LIP_JUDGMENT_ADMISSION,
+    JBA_ONLINE_LIVE_ENABLED,
```

Comment:
I think it shuld be named as "JO" then "JBA" ?

Reply:
Updated the name to JO_ONLINE_LIVE_ENABLED